client/core/trade.go

```diff
@@ -953,11 +952,14 @@ func (t *trackedTrade) isRedeemable(ctx context.Context, match *matchTracker) bo
 			return ready
 		}
 		// If we're the taker, check the confirmations anyway so we can notify.
-		confs, err := t.wallets.fromWallet.SwapConfirmations(ctx, match.MetaData.Proof.TakerSwap,
+		confs, spent, err := t.wallets.fromWallet.SwapConfirmations(ctx, match.MetaData.Proof.TakerSwap,
 			match.MetaData.Proof.Script, match.MetaData.Stamp)
 		if err != nil {
 			t.dc.log.Errorf("error getting confirmation for our own swap transaction: %v", err)
 		}
+		if spent {
+			t.dc.log.Errorf("our (taker) swap for match %s is being reported as spent", match)
```

Comment:
Only thing with these ERR logs is that they will happen routinely in the time between when the counterparty spends your contract and when they notify server and server audits it and server notifies us.  So it doesn't reflect a problem, at least not immediately.